catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/monitor/impl/WorkspaceServiceImpl.java

```diff
@@ -184,24 +213,9 @@ private QueryRequest createQueryRequest(List<Filter> filters) {
    * @return query request
    */
   private QueryRequest createQueryRequest(Filter filter) {
-    return new QueryRequestImpl(new QueryImpl(filter), createProperties());
-  }
-
-  /**
-   * Create the filters for getting workspace metacards based on workspace ids.
-   *
-   * @param workspaceIds set of workspace ids
-   * @return list of filters
-   */
-  private List<Filter> createFilters(Set<String> workspaceIds) {
-    return workspaceIds
-        .stream()
-        .map(
-            id ->
-                filterBuilder.allOf(
-                    filterService.buildMetacardIdFilter(id),
-                    filterService.buildWorkspaceTagFilter()))
-        .collect(Collectors.toList());
+    QueryImpl query = new QueryImpl(filter);
+    query.setPageSize(maxSubscriptions.get());
```

Comment:
❓ If this page size is greater than the internal framework limit it means the subscription size could never be greater than that. Did we want to use the ResultIterator here?

Reply:
To be safe, I switched over to using ResultIterable. I followed the approach from the CswEndpoint. Please come back and double-check my impl.

Reply:
Is `query.setPageSize(maxSubscriptions);` still needed when using ResultIterable?